dev-tools/src/lambdas/dynamoOperations/handler.ts

```diff
@@ -14,26 +14,29 @@ export const handler = async (
   context: Context
 ) => {
   initialiseLogger(context)
-  logger.info(
-    'Function called with following params: ',
-    JSON.stringify(dynamoOperationParams)
-  )
-
   if (!dynamoOperationParams) {
     throw Error('Function called with undefined params')
   }
   if (dynamoOperationParams.params.zendeskId) {
     appendZendeskIdToLogger(dynamoOperationParams.params.zendeskId)
   }
 
+  let result
   switch (dynamoOperationParams.operation) {
     case 'GET':
-      return await dynamoDbGet(dynamoOperationParams.params)
+      result = await dynamoDbGet(dynamoOperationParams.params)
+      logger.info('Sent GetItemCommand to Dynamo')
```

Comment:
These should read something like `GetItemCommand successful` since the function actually completes the operation.